src/Builder/DatagridBuilder.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Sonata\DoctrineMongoDBAdminBundle\Builder;
 
+use Doctrine\ODM\MongoDB\Mapping\ClassMetadataInfo;
```

Comment:
cc @greg0ire wasn't `ClassMetadataInfo` deprecated?

Reply:
The ORM one is.

Reply:
But maybe the one in ODM is too, there is https://github.com/doctrine/mongodb-odm/blob/1.3.x/lib/Doctrine/ODM/MongoDB/Mapping/ClassMetadata.php#L40

Reply:
@alcaeus can you please advise?

Reply:
Correct - this will also be dropped in ODM 2.0.

Reply:
But it's only deprecated from release 1.3 and the current one is 1.2.5. There is no ClassMetadata in current release.

Reply:
Ah right. Let's keep it that way then.

Reply:
> But it's only deprecated from release 1.3 and the current one is 1.2.5.

Technically correct.

> There is no ClassMetadata in current release.

Yes there is: https://github.com/doctrine/mongodb-odm/blob/b47979e8af5830f15790ea731441e4bdf6876d80/lib/Doctrine/ODM/MongoDB/Mapping/ClassMetadata.php#L40

Reply:
Crazy, don't know why I've not seen it. I think I switched my branch faster than my ide has updated. ;)